### PR TITLE
[Magiclysm] Adding more attunement spells (1/3)

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Mods "Adding more attunement spells"
#### Purpose of change
Adding:
Boreal mage "Auroral guard" spell (temporary armor) + item +material for it (it strange for me that ethereal items has non_ethereal materials)
Gaias chosen "Samsara" spell (heal and cure with price of rune)
Sun mage "Pocket sun" spell (temporary light and boil item)
#### Testing
~~I didnt test it yet, cause i did it a long time ago, but ill test if in few hours~~ it works, except fact that sun cant be spawned not in character hands, so i just delete his range
#### Additional context
Some of the changes that was maded by maintainers were lost, like some descriptions errors (sorry for that), so i hope someone can check them and some stuff that was added after content freeze that i didnt notice.
~~Also its not linted yet.~~ linted

### CODE REVIEW

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -856,5 +856,63 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "aura_auroral_guard",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "auroral shell" },
+    "description": "Sturdy shell what wrap your body and gain almost perfect defence.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/items/ethereal_items.json:865]:
```suggestion
    "description": "A sturdy shell that wraps your body, gaining a much better defense.",
```

**Comment by actual-nh** [comment]
I'm not a maintainer, BTW.

File: data/mods/Magiclysm/Spells/attunements/Boreal_Mage.json [hunk 2]
```diff
@@ -48,5 +48,25 @@
     "energy_source": "MANA",
     "damage_type": "cold",
     "max_level": 35
+  },
+  {
+    "id": "borealmage_auroral_guard",
+    "type": "SPELL",
+    "name": { "str": "Auroral Guard" },
+    "description": "Create a sturdy shell that wraps your body, gaining a much better defense.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Boreal_Mage.json:56]:
```suggestion
    "description": "Create a sturdy shell that wraps around your body, gaining a sturdy defense.",
```

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You let your mana spin in the world's life flow, renewing your body as if prematurely reincarnated.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 250000,
+    "final_casting_time": 50000,
+    "casting_time_increment": -6250,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:13]: the casting time for this is too high. i don't see a reason to have the decrement, so i'd probably do it this way:
```suggestion
    "base_casting_time": 60000,
```

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "id": "gaiaschosen_samsara",
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You let your mana spin in the world's life flow, renewing your body as if prematurely reincarnated.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:7]:
```suggestion
    "description": "You tie your leylines to the world, renewing your flesh as if reborn.",
```

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "components": "spell_components_samsara",
+    "difficulty": 8,
+    "min_damage": -20,
+    "max_damage": -200,
+    "damage_increment": -4.5,
+    "max_level": 40,
+    "extra_effects": [ { "id": "gaiaschosen_samsara_cureall", "hit_self": true } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ]
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:23]:
```suggestion
    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "LOUD" ]
```
this isn't really what this flag's for

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  {
+    "id": "gaiaschosen_samsara_cureall",
+    "type": "SPELL",
+    "name": "Samsara Cureall effect",
+    "description": "Gives you a cure effect.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "effect_str": "cureall",
+    "shape": "blast"
+  }
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:34]:
```suggestion
  }
```
just put this effect in the main spell.

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 7]
```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "id": "gaiaschosen_samsara",
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You let your mana spin in the world's life flow, renewing your body as if prematurely reincarnated.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:9]:
```suggestion
    "effect": "attack",
    "effect_str": "cureall",
```

File: data/mods/Magiclysm/Spells/attunements/Sun_Mage.json [hunk 8]
```diff
@@ -29,5 +29,27 @@
     "max_level": 35,
     "base_casting_time": 350,
     "energy_source": "MANA"
+  },
+  {
+    "id": "sunmage_pocket_sun",
+    "type": "SPELL",
+    "name": "Pocket Sun",
+    "description": "You create a stable gap in the fabric of reality near the sun, through which you can draw on its light and temperature.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Sun_Mage.json:37]:
```suggestion
    "description": "You create a stable portal in the fabric of reality to the sun, through which you can draw on its light and temperature.",
```

File: data/mods/Magiclysm/Spells/attunements/Sun_Mage.json [hunk 9]
```diff
@@ -29,5 +29,27 @@
     "max_level": 35,
     "base_casting_time": 350,
     "energy_source": "MANA"
+  },
+  {
+    "id": "sunmage_pocket_sun",
+    "type": "SPELL",
+    "name": "Pocket Sun",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Sun_Mage.json:36]: are you _sure_ this spell is strong enough to take up one of the 2 spell slots for the attunement?

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 10]
```diff
@@ -856,5 +856,67 @@
...
+    "material_thickness": 1.7,
+    "environmental_protection": 8,
+    "flags": [ "ONLY_ONE", "OVERSIZE", "PERSONAL", "STURDY" ]
+  },
+  {
+    "id": "pocket_sun",
+    "type": "TOOL",
+    "category": "tools",
+    "name": { "str": "Pocket Sun" },
+    "description": "A small, capsule-shaped field containing a gap in reality that lets through solar energy.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json:887]:
```suggestion
    "description": "A small, capsule-shaped field containing a stable portal to the sun.",
```

File: data/mods/Magiclysm/materials.json [hunk 11]
```diff
@@ -237,5 +237,22 @@
     "name": "Mana",
     "copy-from": "pseudo_fuel",
     "fuel_data": { "energy": 500, "perpetual": true }
+  },
+  {
+    "type": "material",
+    "id": "concentrated_mana",
+    "name": "Concentrated Mana",
+    "density": 38,
+    "reinforces": true,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/materials.json:246]:
```suggestion
    "reinforces": false,
```
and isn't there also a "no repair" thing? there's no way you can repair this.

**Review by KorGgenT — COMMENTED** [review]
Generally okay, but i really need to hear about the sun mage spell. it's very important to me that all of the attunement spells are sufficiently useful and powerful that people don't get the attunement and say "oh man, i wish it didn't have _this_"

File: data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json [hunk 12]
```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "gaiaschosen_samsara",
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You tie your leylines to the world, renewing your flesh as if reborn.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json:7]:
```suggestion
    "description": "You tie your ley-lines to the world's, renewing your flesh as if reborn.",
```

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 13]
```diff
@@ -856,5 +856,73 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "aura_auroral_guard",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "auroral shell" },
+    "description": "A sturdy shell that wraps your body, gaining a much better defense.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/items/ethereal_items.json:865]:
```suggestion
    "description": "A sturdy shell that wraps your body, gaining a sturdy defense.",
```
To match the spell description.

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 14]
```diff
@@ -856,5 +856,73 @@
...
+      { "type": "firestarter", "moves": 10, "moves_slow": 100 }
+    ],
+    "flags": [ "LIGHT_700", "TRADER_AVOID" ]
+  },
+  {
+    "id": "pocket_sun_cover",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "Covered Pocket Sun" },
+    "description": "A small, capsule-shaped field containing a stable portal to the sun. It is currently covered.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/items/ethereal_items.json:910]:
```suggestion
    "description": "A small, capsule-shaped field containing a stable portal to the sun.  It is currently covered.",
```

**Review by actual-nh — COMMENTED** [review]
I also note a problem at the bottom of the "Files changed" screen.

**Pull request merged**